nevergrad/instrumentation/discretization.py

```diff
@@ -42,6 +42,13 @@ def inverse_threshold_discretization(indexes: List[int], arity: int = 2) -> Arra
     return scipy.stats.norm.ppf(indexes_arr * pdf_bin_size + (pdf_bin_size / 2))  # type: ignore
 
 
+def noisy_inverse_threshold_discretization(indexes: List[int], arity: int = 2, gen: Any = None) -> ArrayLike:
```

Comment:
Be more explicit on what this discretization does, it is not that clear.

Reply:
Not sure of it, since there is supposed to have no order relation in cGA discrete algorithm

Reply:
The name was chosen by @jrapin and i believe it is correct --- it is specified "inverse" here.

Reply:
Personally I don't find the function really intuitive and its name too. Be more explicit on what does this function

Reply:
Done.